app/src/main/java/com/codesoom/assignment/application/TaskService.java

```diff
@@ -24,18 +24,16 @@ public Task getTask(Long id) {
     }
 
     public Task createTask(Task source) {
-        Task task = new Task();
-        task.setId(generateId());
-        task.setTitle(source.getTitle());
-
+        Long id = generateId();
+        Task task = new Task(id, source.getTitle());
```

Comment:
```suggestion
        Task task = new Task(generateId(), source.getTitle());
```

바로 밑에서 한번 사용될 것이며, 메소드 이름에 이미 무엇을 하는지 적혀있기에 굳이 변수로 지정하지 않아도 될 것 같아요!